docs/source/guides/index.md

```diff
@@ -0,0 +1,9 @@
+# Guides
+
+```{toctree}
+:caption: Guides
+:hidden:
+
+./model-configuration
+./releasing-to-PyPI
```

Comment:
@ashwinvaidya17 @sovrasov I'm not sure if we need to include `PyPI releasing guide` to public doc. What do you think?

Reply:
I kept the existing docs as is. Ideally we shouldn't have this guide in the first place. Releasing to PyPI can be easily be automated with github actions.

Reply:
Right, that's only for internal usage